src/components/ServerConnections.js

```diff
@@ -40,10 +39,10 @@ class ServerConnections extends ConnectionManager {
 
         Events.on(this, 'localusersignedout', (_e, logoutInfo) => {
             setUserInfo(null, null);
+            // Reset cached views
+            document.dispatchEvent(new CustomEvent('skinunload', {}));
             // Ensure the updated credentials are persisted to storage
             credentialProvider.credentials(credentialProvider.credentials());
-            // Reset the query cache
-            queryClient.resetQueries();
```

Comment:
Let's move this back since it wasn't the root of the issue and we can keep this reset logic in one place.

Reply:
the `queryClient.resetQueries()` doesn't rest the `tanstack cache`. moving the reset logic to dashboard page > logout it work as expected

Reply:
Before

https://github.com/jellyfin/jellyfin-web/assets/32230989/b27b902a-5cc9-4b29-aeea-fe78b58000f2

After

https://github.com/jellyfin/jellyfin-web/assets/32230989/c6723364-f4aa-471b-958a-6a7e294fc94a

Reply:
https://github.com/jellyfin/jellyfin-web/pull/5636